RecoTracker/MkFitCore/interface/IterationConfig.h

```diff
@@ -140,23 +129,49 @@ namespace mkfit {
 
   class IterationConfig {
   public:
-    using partition_seeds_foo = void(const TrackerInfo &,
-                                     const TrackVec &,
-                                     const EventOfHits &,
-                                     IterationSeedPartition &);
+    // Called directly.
+    using clean_seeds_cf = int(TrackVec &, const IterationConfig &, const BeamSpot &);
+    using clean_seeds_func = std::function<clean_seeds_cf>;
+    // Called from MkBuilder::find_tracks_load_seeds().
+    using partition_seeds_cf = void(const TrackerInfo &,
+                                    const TrackVec &,
+                                    const EventOfHits &,
+                                    IterationSeedPartition &);
+    using partition_seeds_func = std::function<partition_seeds_cf>;
+    // Passed to MkBuilder::filter_comb_cands().
+    using filter_candidates_cf = bool(const TrackCand &, const MkJob &);
+    using filter_candidates_func = std::function<filter_candidates_cf>;
+    // Called directly.
+    using clean_duplicates_cf = void(TrackVec &, const IterationConfig &);
+    using clean_duplicates_func = std::function<clean_duplicates_cf>;
 
     int m_iteration_index = -1;
     int m_track_algorithm = -1;
 
     bool m_requires_seed_hit_sorting = false;
-    bool m_requires_quality_filter = false;
-    bool m_requires_dupclean_tight = false;
 
     bool m_backward_search = false;
     bool m_backward_drop_seed_hits = false;
 
     int m_backward_fit_min_hits = -1;  // Min number of hits to keep when m_backward_drop_seed_hits is true
 
+    // seed cleaning params
```

Comment:
These should all definitely be configurable and not magic numbers.

Reply:
these are defaults; the needed values are configurable.